markdown/bestpractices/managing_resources_bestpractices.html.md.erb

```diff
@@ -110,11 +110,14 @@ ALTER RESOURCE queue1 WITH (VSEG_RESOURCE_QUOTA='mem:2gb');
 If there are only occasional queries on Parquet tables with a large page size, use a statement level specification instead of altering the resource queue. For example:
 
 ```sql
-SET HAWQ_RM_STMT_NVSEG=10;
-SET HAWQ_RM_STMT_VSEG_MEMORY='2gb';
-query1;
-SET HAWQ_RM_STMT_NVSEG=0;
+postgres=# SET hawq_rm_stmt_nvseg=10;
+postgres=# SET hawq_rm_stmt_vseg_memory='2gb';
+postgres=# query1;
+postgres=# SET hawq_rm_stmt_nvseg=0;
+postgres=# SET
```

Comment:
I'm pretty sure this SET command isn't actually a command to type at the postgres=# prompt, but an output from psqsl.  See the example at the end of this PR (each SET <something> command results in psql showing SET).  You should make both code blocks similar in this regard.